test/smoke/src/areas/positron/notebook/notebookCreate.test.ts

```diff
@@ -12,53 +12,40 @@ describe('Notebooks #pr #web #win', () => {
 	setupAndStartApp();
 
 	describe('Python Notebooks', () => {
+		let app: Application;
 
 		before(async function () {
-
-			await PositronPythonFixtures.SetupFixtures(this.app as Application);
-
+			app = this.app as Application;
+			await PositronPythonFixtures.SetupFixtures(app);
+			await app.workbench.positronLayouts.enterLayout('notebook');
 		});
 
-		after(async function () {
+		beforeEach(async function () {
+			await app.workbench.positronNotebooks.createNewNotebook();
+			await app.workbench.positronNotebooks.selectInterpreter('Python Environments', process.env.POSITRON_PY_VER_SEL!);
+		});
```

Comment:
might want to wrap this in a screenshot try-catch since it won't be part of the trace and it can be error prone.

Reply:
why wouldn't it be part of the trace? i'm almost certain only `before` and `after` are not part of trace or retries.

Reply:
FYI, I did a local run and intentionally failed test in the beforeEach and confirmed that trace is present. 👍